docs/api.rst

```diff
@@ -4,101 +4,152 @@
+function-based API. This page outlines the stable API. For the legacy functionality, see
+:doc:`Legacy API  <legacy_api>`.
+
+Managing morphological elements
+-------------------------------
+.. _elements:
+
+Momepy allows creation of a small subset of bespoke morphological geometric features.
 
 .. autosummary::
-   :toctree: generated/
+   :toctree: api/
 
-   Blocks
-   buffered_limit
-   COINS
+   morphological_tessellation
+   enclosed_tessellation
    enclosures
-   get_network_id
+   generate_blocks
+
+Additionally, it contains tools supporting these.
+
+.. autosummary::
+   :toctree: api/
+
+   buffered_limit
+   verify_tessellation
+
+And tools linking various elements together.
+
+.. autosummary::
+   :toctree: api/
+
+   get_nearest_street
    get_network_ratio
    get_node_id
-   Tessellation
 
-dimension
----------
+Measuring dimension
+-------------------
+
+A set of functions to measure dimensions of geometric elements:
 
 .. autosummary::
-   :toctree: generated/
-
-   Area
-   AverageCharacter
-   CourtyardArea
-   CoveredArea
-   FloorArea
-   LongestAxisLength
-   Perimeter
-   PerimeterWall
-   SegmentsLength
-   StreetProfile
-   Volume
-   WeightedCharacter
-
-shape
------
+   :toctree: api/
+
+   courtyard_area
+   floor_area
+   longest_axis_length
+   perimeter_wall
+   street_profile
+   volume
+   weighted_character
+
+Measuring shape
+---------------
+
+A set of functions to measure shape of geometric elements:
 
 .. autosummary::
-  :toctree: generated/
-
-  CentroidCorners
-  CircularCompactness
-  CompactnessWeightedAxis
-  Convexity
-  Corners
-  CourtyardIndex
-  Elongation
-  EquivalentRectangularIndex
-  FormFactor
-  FractalDimension
-  Linearity
-  Rectangularity
-  ShapeIndex
-  SquareCompactness
-  Squareness
-  VolumeFacadeRatio
-
-spatial distribution
---------------------
+   :toctree: api/
+
+   centroid_corner_distance
+   circular_compactness
+   compactness_weighted_axis
+   convexity
+   corners
+   courtyard_index
+   elongation
+   equivalent_rectangular_index
+   facade_ratio
+   form_factor
+   fractal_dimension
+   linearity
+   rectangularity
+   shape_index
+   square_compactness
+   squareness
+
+Measuring spatial distribution
+------------------------------
+
+A set of functions to measure spatial distribution of geometric elements:
+
 .. autosummary::
-  :toctree: generated/
-
-  Alignment
-  BuildingAdjacency
-  CellAlignment
-  MeanInterbuildingDistance
-  NeighborDistance
-  NeighboringStreetOrientationDeviation
-  Neighbors
-  Orientation
-  SharedWalls
-  SharedWallsRatio
-  StreetAlignment
-
-intensity
----------
+   :toctree: api/
+
+   alignment
+   building_adjacency
+   cell_alignment
+   mean_interbuilding_distance
+   neighbor_distance
+   neighbors
+   orientation
+   shared_walls
+   street_alignment
+
+Measuring intensity
+-------------------
+
+A set of functions to measure intensity characters:
```

Comment:
Should this be "characteristics"?

Reply:
These are "morphometric characters", which is a term used in literature.